Sources/SwiftKuery/ConnectionPoolConnection.swift

```diff
@@ -145,6 +179,8 @@ public class ConnectionPoolConnection: Connection {
         connection.execute(raw, parameters: parameters, onCompletion: onCompletion)
     }
     
+    // MARK: Prepared statements
+
     /// Prepare statement.
```

Comment:
Create a prepared statement from the passed in query.
(Again there is another of these below).